src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/Courses/RelatedOccupationResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.Courses;
+
+[ExcludeFromCodeCoverage]
+public class RelatedOccupationResponse
```

Comment:
If you add RelatedOccupation under domain name space then you do not need two duplicate classes